index.js

```diff
@@ -1,2 +1,4 @@
 const test = "toto";
 console.log(test);
```

Comment:
- eslint: Combine this with the previous 'const' statement.
- eslint: 'toto' is assigned a value but never used.